JPS_Ontology/KBTemplates/TBox/OntoSewage.csv

```diff
@@ -0,0 +1,169 @@
+hasClass,Object Property,,,Cover,CoverClass,,A relationship between a cover and cover class,https://www.theworldavatar.com/kg/ontosewage/,has class
+hasShaft,Object Property,,,https://saref.etsi.org/saref4watr/Manhole,Shaft,,A relationship between a manhole and shaft,https://www.theworldavatar.com/kg/ontosewage/,has shaft
+hasDepth,Object Property,,,Shaft,http://www.ontology-of-units-of-measure.org/resource/om-2/Depth,,A relationship between a shaft and depth,https://www.theworldavatar.com/kg/ontosewage/,has depth
+hasFlume,Object Property,,,https://saref.etsi.org/saref4watr/Manhole,Flume,,A relationship between a manhole and flume,https://www.theworldavatar.com/kg/ontosewage/,has flume
+hasNodeType,Object Property,,,https://saref.etsi.org/saref4watr/Manhole,NodeType,,A relationship between a manhole and its type,https://www.theworldavatar.com/kg/ontosewage/,has node type
+hasConnectionType,Object Property,,,https://saref.etsi.org/saref4watr/Manhole,ConnectionType,,A relationship between a manhole and connection type,https://www.theworldavatar.com/kg/ontosewage/,has connection type
+isAssociatedWith,Object Property,,,https://schema.org/location,AssociatedInfrastructure,,A relationship between a location and associated infrastructure,https://www.theworldavatar.com/kg/ontosewage/,is associated with
+http://semanticscience.org/resource/SIO_000061,Object Property,,,https://schema.org/location,http://rdfs.co/juso/Street UNION http://rdfs.co/juso/District,,,http://semanticscience.org/ontology/sio.owl,
+hasCoordinates,Object Property,,,https://schema.org/location,https://schema.org/GeoCoordinates,,A relationship between a location and Geo coordinates,https://www.theworldavatar.com/kg/ontosewage/,has coordinates
+hasElevation,Object Property,,,https://schema.org/location,https://schema.org/Elevation,,A relationship between a location and elevation,https://www.theworldavatar.com/kg/ontosewage/,has elevation
+hasElevationAboveSeaLevel,Object Property,,,https://schema.org/Elevation,http://www.ontology-of-units-of-measure.org/resource/om-2/Height,,A relationship which shows the elevation above see level of a location,https://www.theworldavatar.com/kg/ontosewage/,has elevation above sea level
+hasElevationReference,Data Property,,,https://schema.org/Elevation,String,,A relationship which shows the elevation reference,https://www.theworldavatar.com/kg/ontosewage/,has elevation reference
+hasElevationAccuracy,Data Property,,,https://schema.org/Elevation,String,,A relationship which shows the elevation accuracy,https://www.theworldavatar.com/kg/ontosewage/,has elevation accuracy
+componentID,Data Property,,,SewerageComponent,String,,sewerage component ID,https://www.theworldavatar.com/kg/ontosewage/,component ID
+hasSewagePlantID,Data Property,,,SewerageRecords,Integer,,sewerage plant ID,https://www.theworldavatar.com/kg/ontosewage/,has sewage plant ID

```

Comment:
I am (still) not convinced that this is the right way to do it. Potentially that's good enough for now, but ideally one would like to have a relationship along the lines of `sewerage component` - `feeds into` - `sewage treatment plant` and then several instances of `sewage treatment plant` instead of having a "random" Integer associated with the `sewerage component` which only makes sense after being looked up its meaning in some table

Reply:
If we change it to this, do we have any instance for that in the Data? Could you please explain more about this?